vulnerability_report.py

```
@@ -0,0 +1,432 @@
+        "Multiple JSON files with Dependabot alerts needs to be provided",
+    )
+
+    return parser
+
+
+def check_args(args: Namespace) -> None:
+    """Check the validity of all command line arguments."""
+    if args.generate_graphs:
+        # when graphs should be generated, output format need to be specified too
+        if not args.svg_output and not args.png_output:
+            raise "Graphs generation was requested: you need to select the PNG and/or SVG output."
```

Ruff (0.15.18)

[warning] Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)